verifiers/verifier_regression_test.go

```diff
@@ -0,0 +1,132 @@
+//go:build regression
+
+package verifiers
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"hash"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	sigstoreTuf "github.com/sigstore/sigstore-go/pkg/tuf"
+	serrors "github.com/slsa-framework/slsa-verifier/v2/errors"
+	"github.com/slsa-framework/slsa-verifier/v2/options"
+)
+
+const TEST_DIR = "../cli/slsa-verifier/testdata"
```

Comment:
It might be worth creating a new `testdata/` folder in the folder this test file is in and copying the subset of test data you need from `.../cli/slsa-verifier/testdata` into it.

Having this test depend directly on the test data in `.../cli/slsa-verifier/testdata` could make maintenance more difficult in the future e.g. if someone wants to change the test data for the cli or move the `cli/slsa-verifier` folder it will break these tests. It might seem a bit wasteful to have two separate copies of the data but as the Go proverb goes "A little copying is better than a little dependency." (https://go-proverbs.github.io/).

Reply:
added.